portable/GCC/ARM_CM23_NTZ/non_secure/port.c

```diff
@@ -185,9 +186,11 @@ typedef void ( * portISR_t )( void );
 #define portMPU_MAIR0_REG                       ( *( ( volatile uint32_t * ) 0xe000edc0 ) )
 #define portMPU_MAIR1_REG                       ( *( ( volatile uint32_t * ) 0xe000edc4 ) )
 
-#define portMPU_RBAR_ADDRESS_MASK               ( 0xffffffe0 ) /* Must be 32-byte aligned. */
+#define portMPU_RBAR_ADDRESS_MASK               ( 0xffffff00 ) /* Must be 32-byte aligned. */
 #define portMPU_RLAR_ADDRESS_MASK               ( 0xffffffe0 ) /* Must be 32-byte aligned. */
+#define portMPU_RASR_LENGTH_MASK                ( 0x1F )       /* Must be [7:31]. */
```

Comment:
Why not just make this `0x3E` and avoid the later shifting in `portEXTRACT_REGION_LENGTH_FROM_RASR`

Reply:
Both will work here however it is done this way more to align with conventions.